applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py

```diff
@@ -141,18 +142,21 @@ def AddVariables(self):
         # Add displacements.
         self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.DISPLACEMENT)
         self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.REACTION)
-        # Add specific variables for the problem conditions.
-        self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.POSITIVE_FACE_PRESSURE)
-        self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.NEGATIVE_FACE_PRESSURE)
-        self.main_model_part.AddNodalSolutionStepVariable(StructuralMechanicsApplication.POINT_LOAD)
-        self.main_model_part.AddNodalSolutionStepVariable(StructuralMechanicsApplication.LINE_LOAD)
-        self.main_model_part.AddNodalSolutionStepVariable(StructuralMechanicsApplication.SURFACE_LOAD)
-        self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.VOLUME_ACCELERATION)
+        # Add specific variables for the problem (rotation dofs).
         if self.settings["rotation_dofs"].GetBool():
-            # Add specific variables for the problem (rotation dofs).
             self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.ROTATION)
             self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.REACTION_MOMENT)
-            self.main_model_part.AddNodalSolutionStepVariable(StructuralMechanicsApplication.POINT_MOMENT)
+        if (self.settings["retrocompatibility_on_loads"].GetBool() is True):
```

Comment:
and btw... what does it mean "retrocompatibility on loads"?

I do not agree that the default is a constant one. 
For example VOLUMETRIC_ACCELERATION is an excellent example of what you may well want on the nodes, since doing it nodal allows to work in a inertial system (think for example of something rotating very fast)

Reply:
@RiccardoRossi I think that make it optional is the key. You can consider any behavior, but there is not any additional load variable charge because of that

Reply:
In any case, @RiccardoRossi you can change the value without being historical the variable. The only reason to use historical variables is to compute something that requires know the value in the previous steps, your example doesn't fit